### PR TITLE
[AEA-828] test multiplexer disconnected on agent process termination

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

test multiplexer disconnected on agent process termination
this is just tests and one is failing.
## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by solarw** [comment]
there is a test that raises a problem and the solution is in  https://github.com/fetchai/agents-aea/pull/1731

so will be ready to merge after PR above will be merged.

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**
